### PR TITLE
1.0.1 Doc Changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Rewrote README.md
- Rewrote setup.md
- Rewrote configuration instructions and usage

#### Descriptions:
_Add PR descriptions here_

#### Fix:
_Add fix/changes here_

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -1,57 +1,143 @@
...
+| Icon for Record View                       | Select a field of type `Lookup` using which to display the icons representing asset records. For more information, refer to [set up modules to use icons that represent records](#setting-up-modules-to-use-icons-for-records). |
+| Show Correlation Edges                     | Select this option to add visual lines on the view to depict correlations between records from the same module grouped at various levels. |
+| Record Assignment (Default Filter)         | Select to display records that are assigned to the logged-in user or to display all records based on the defined filters.                 |
+| Assignment Fields                          | Select the field based on which records are assigned for the select module.                                                               |
+| Filter Criteria                            | Define the filter criteria using which to filter the data retrieved by this widget.                                                       |
+
```
Comment (cs-ankit-gupta) [inline docs/setup.md:32]: One of the field "Title for Record View" is missing

File: docs/setup.md [hunk 2]
```diff
@@ -1,57 +1,143 @@
...
+3. Drag-and-drop an icon file. Alternatively, you can browse to the icon file and select to upload it.
+
+4. Enter the description of the asset type in the **Description** field.
+
+5. Click **Save** to save the record.
+
+Add this widget in any of the views to see the uploaded icons.
+
+| [Usage](./docs/usage.md) |
```
Comment (cs-ankit-gupta) [inline docs/setup.md]: link is broken

File: docs/usage.md [hunk 3]
```diff
@@ -1,40 +1,97 @@
...
+![Displaying the Alert Distribution widget on an Asset record](./res/RecordDistributionModuleView.png)
 
-<img src="https://raw.githubusercontent.com/fortinet-fortisoar/widget-record-distribution/release/1.0.0/docs/media/RecordDistributionModuleView.png" alt="Displaying the Alert Distribution widget on a Asset record" style="border: 1px solid #A9A9A9; border-radius: 4px; padding: 10px; display: block; margin-left: auto; margin-right: auto;">
+<table>
+    <td><strong>NOTE</strong></td><td>If the asset record does not have any linked alerts, then the <strong>Record Distribution</strong> widget displays the following message:<br><br>
+    <strong>No records found!</strong></td>
+</table>
 
-**Note**: If the asset record does not have any linked alerts, then the Record Distribution widget will display a '**No records found!**' message.
+| [Installation](./docs/setup.md#installation) | [Configuration](./docs/setup.md#configuration) |
```
Comment (cs-ankit-gupta) [inline docs/usage.md]: link is broken

File: docs/setup.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md:32]: image

@cs-ankit-gupta: Please provide inputs.

File: docs/usage.md [hunk 4]
```diff
@@ -1,40 +1,97 @@
...
-<img src="https://raw.githubusercontent.com/fortinet-fortisoar/widget-record-distribution/release/1.0.0/docs/media/editingRecordDistribution.png" alt="Configuring the Record Distribution Widget" style="border: 1px solid #A9A9A9; border-radius: 4px; padding: 10px; display: block; margin-left: auto; margin-right: auto;">
+- Add visual lines to identify correlations between grouped records from the same module at different levels.
+
+## Record Distribution Widget
+
+<table>
+    <td><strong>NOTE</strong></td><td>Refer to the section <a href="./setup.md#setting-up-modules-to-use-icons-for-records">Setting up Modules to use Icons for Records</a> in <em>Configuration</em> instructions before integrating the widget in your workflows.</td>
+</table>
+
+1. Edit a module's (e.g. *Dashboard*) view template and select **Add Widget** button.
```
Comment (cs-vikas-shelar) [inline docs/usage.md]: IMO it should be like 'Edit a dashboard or module record's view template'.

File: docs/setup.md (hunk 1 above)
Comment (cs-vikas-shelar) [reply docs/setup.md:32]: @cs-abhishek-shukla This field represents the title displayed below each icon in the Record distribution widget. Users can configure which title widget can show, like Name, IP address, etc.

**Review by cs-vikas-shelar — CHANGES_REQUESTED** [review]
Minor comment added. Other changes looks good to me.

**Review by cs-ankit-gupta — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Release Information
 
-- **Version**: 1.0.0 <sup>Preview</sup>
+- **Version**: 1.0.1 <sup>Preview</sup>
```
Comment (cs-suyog-jinturkar) [inline README.md]: Why preview? @cs-ankit-gupta

File: README.md [hunk 6]
```diff
@@ -12,13 +12,20 @@
 
 - [Release Notes](./widget/release_notes.md)
 
-**Note**: Preview releases are a beta release. This means that release is intended to get feedback and might not be best suited for production level deployments. The functionality might change in backward-incompatible ways or have limited support. A beta release is not subject to any SLA, Quality Assurance or deprecation policy. Feature availability and support for preview releases will continue to improve as the solution/feature matures.
+
+<table>
+    <td><strong>NOTE</strong></td><td>Preview releases are a beta release, is intended to get feedback and might not be best suited for production level deployments. The functionality might change in backward-incompatible ways or have limited support. A beta release is not subject to any SLA, Quality Assurance or deprecation policy. Feature availability and support for preview releases will continue to improve as the solution/feature matures.</td>
```
Comment (cs-suyog-jinturkar) [inline README.md]: If we happen to remove preview tag then, we will need to remove this text as well

File: docs/usage.md [hunk 7]
```diff
@@ -1,40 +1,97 @@
...
+The **Record Distribution** widget helps visualise records or items based on a specified grouping criterion. For example, viewing the distribution of assets by purdue levels, or the distribution of alerts by type, severity, or other criteria.
 
-- Ability to group records using a picklist, say "Levels" in the "Asset" module. The benefit of having a grouped view is that you can view all of the records together based on the grouping. In the case of our example, it would be view assets at each level. Additionally, it also provides a visual depiction of each record node in the group.
-- Ability to add this view on a dashboard, report, or within the detail view of a module's record. 
+You can group records using a picklist, say *Levels* in the **Assets** module. With such a criterion you can view all of the assets together and at each level. Additionally, it also provides a visual depiction of each record node in the group.
 
-An example of using the Record Distribution widget would be to display the assets grouped by their levels.
+You can add this view to a dashboard, report, or within the detailed view of a module's record.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:10]: This info can be added for all widgets going forward in overview section like we mention about version info. The title for the same could be compatibility

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added few minor comments. Rest LGTM

**Comment by cs-vikas-shelar** [comment]
The documentation looks good. Only need to finalize and address comments from Suyog.

**Review by cs-vikas-shelar — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
 # Release Information
 
-- **Version**: 1.0.0 <sup>Preview</sup>
+- **Version**: 1.0.1
 
 - **Certified**: No
```
Comment (cs-ankit-gupta) [inline README.md]: Certified Yes

**Pull request merged**